articles/libraries/lock/v10/lock-passwordless-migration-guide.md

```diff
@@ -0,0 +1,147 @@
+---
+section: libraries
+toc: true
+description: Migration Guide for lock-passwordless to Lock 10 with Passwordless Mode
+---
+# Migration Guide for lock-passwordless to Lock 10 with Passwordless Mode
+
+The following instructions assume you are migrating from **lock-passwordless** to the latest **Lock 10** using the **Passwordless** mode.
+
+The [lock-passwordless](https://github.com/auth0/lock-passwordless) widget was previously a standalone library, separate from [Lock 10](https://auth0.com/docs/libraries/lock/v10). Now, you can migrate your apps to use the newest Lock Passwordless, which is integrated directly into Lock 10. Lock 10 with Passwordless Mode is the latest method by which to quickly and simply deploy a login widget for passwordless authentication in your apps.
+
+The goal of this migration guide is to provide you with all of the information that you will need to update your **lock-paswordless** installation to **Lock 10** (using Lock 10's **Passwordless Mode**). 
+
+First, you will need to remove **lock-passwordless** from your project, and instead include the [latest release version of Lock 10](https://github.com/auth0/lock/releases). Beyond that, you will then need to take a careful look at each of the sections in this migration guide in order to find out which changes you will need to make to your implementation.
+
+Of particular importance will be the initialization of `Auth0LockPasswordless`, your calls to Lock methods (now the same methods as used in Lock 10), and also your previously implemented customization options, which will need inspected and changed to use the corresponding options for Lock 10.
+
+## General Changes and Additions
+
+### Importing Auth0LockPasswordless
+
+If you're loading from the CDN, you can still use `Auth0LockPasswordless`. The difference is that you'll provide options in the constructor, like we do with `Auth0Lock`:
+
+#### Using the CDN
+
```

Comment:
For all the Before/After snippets I would use this UI (where Legacy/OIDC Conformant would be Before/After):

![image](https://user-images.githubusercontent.com/11715799/30695008-2520bc3c-9edf-11e7-8e3f-4748873c5fb4.png)

You can find the html to build this here https://auth0.com/docs/api-auth/tutorials/adoption/authorization-code